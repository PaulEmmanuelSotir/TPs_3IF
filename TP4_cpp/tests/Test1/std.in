mauvaise_commande1...
EN_ERROR_MESS
mauvaise_commande2...
EXIT
