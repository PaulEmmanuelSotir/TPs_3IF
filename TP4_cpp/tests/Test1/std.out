ERR
OK
Error: Invalid command
